OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -250,13 +253,19 @@ Action InitDisplayPanel(Widget panel)
 				"Windowed" : ds.Mode == WindowMode.Fullscreen ? "Fullscreen (Legacy)" : "Fullscreen";
 
 			var modeChangesDesc = panel.Get("MODE_CHANGES_DESC");
-			modeChangesDesc.IsVisible = () => ds.Mode != WindowMode.Windowed && ds.Mode != OriginalGraphicsMode;
+			modeChangesDesc.IsVisible = () => ds.Mode != WindowMode.Windowed && (ds.Mode != OriginalGraphicsMode ||
+				ds.VideoDisplay != OriginalVideoDisplay);
 
 			var statusBarsDropDown = panel.Get<DropDownButtonWidget>("STATUS_BAR_DROPDOWN");
 			statusBarsDropDown.OnMouseDown = _ => ShowStatusBarsDropdown(statusBarsDropDown, gs);
 			statusBarsDropDown.GetText = () => gs.StatusBars == StatusBarsType.Standard ?
 				"Standard" : gs.StatusBars == StatusBarsType.DamageShow ? "Show On Damage" : "Always Show";
 
+			var displaySelectionDropDown = panel.Get<DropDownButtonWidget>("DISPLAY_SELECTION_DROPDOWN");
+			displaySelectionDropDown.OnMouseDown = _ => ShowDisplaySelectionDropdown(displaySelectionDropDown, ds);
+			var displaySelectionTextTransform = new CachedTransform<int, string>(i => "Display {0}".F(i + 1));
+			displaySelectionDropDown.GetText = () => displaySelectionTextTransform.Update(ds.VideoDisplay);
```

Comment:
Add `displaySelectionDropDown.IsDisabled = () => Game.Renderer.DisplayCount < 2;` to disable the dropdown if there is only a single display available.

I also just noticed that these lines are sitting between the status bar and target line dropdowns - can you please move them up to before `statusBarsDropDown`?